src/app/drive/components/DriveExplorer/DriveExplorerList/DriveExplorerList.tsx

```diff
@@ -446,6 +446,8 @@ const DriveExplorerList: React.FC<DriveExplorerListProps> = memo((props) => {
           />
         )}
         <ShareWithTeamDialog item={props.selectedItems[0]} roles={roles} />
+        {/* TODO: Remove this div when styles works fine from internxt/ui */}
+        <div className={'left-5 border-primary/5 '} />
```

Comment:
Have you left a comment in the UI library to point out that we are going to have to add this until the styling issue has been resolved?

Reply:
No, but I added a task for this. Its included in the release v2.9.0